### DISCUSSION TITLE
Clarification required on lern landing page for compass

### DISCUSSION THREAD

**Discussion by pavankumar0408** [Sunbird-Ed/Community#604] [General Discussion]
In compass, we have lern page where user can view that page only after login. In sunbird we have explore page and course page component, when we inspect that for viewing courses seems like using explore page component instead of course page component. And also got to know that this explore page component using without login by user.

In our case lern lading page can be viewed only after login by user and this page will have features related to courses, browse by competency, topics, providers and others (screenshot attached). For creating this page which existing component can be used from the portal.

![image](https://github.com/Sunbird-Ed/Community/assets/131791648/94bd8ce9-8f27-4686-b7d8-8ee3f5a5f6d5)

**Comment by princegupta1131** [comment]
Hi, @pavankumar0408 In the ED portal explore page is accessible for logged-in & non-logged-in users. 
if you are non-logged-in then route will be **/explore**  & for logged -in  routes will be **/resources.**
if you don't want the content to show for non-logged-in users. please refer to the explore route logic implementation.

CC:@rajeevsathish

**Reply by pavankumar0408** [reply]
@princegupta1131 , Thanks for the details. Could you also please confirm where this course page component is being used.

**Reply by princegupta1131** [reply]
Hi @pavankumar0408 could please debug and verify this?
CC:@rajeevsathish

**Reply by pavankumar0408** [reply]
Will do that. If I need any help will ping you. As of now thanks for the info that you have provided.
